modules/codemod/lib/v13.2/spec/migrateDepthTokens.spec.ts

```diff
@@ -25,6 +25,24 @@ describe('Canvas Kit Tokens > Canvas Tokens v2', () => {
       expectTransform(input, expected);
     });
 
+    it('should not transform depth token to object with boxShadow property if variables are used', () => {
+      const input = stripIndent`
+          import { depth } from "@workday/canvas-kit-react/tokens";
+
+          const size = '1';
+
+          const newDepth = depth[size];
+        `;
+
+      const expected = stripIndent`
+          const size = '1';
+
+          const newDepth = depth[size];
```

Comment:
Question: If `depth` was previously defined in the import statement, and that import is removed, what is now defining `depth`?